server/database/migrations/2019_04_11_180910_add_trigger_before_insert_tabla_clientes.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddTriggerBeforeInsertTablaClientes extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        DB::unprepared('CREATE TRIGGER insert_clientes BEFORE INSERT ON clientes
```

Comment:
Se podría establecer el valor por defecto en las columnas. Usar un trigger es demasiado.

Reply:
pasa que Doctrine no soporta el campo bit para hacer el alter, y como el trigger esta igual por el fecha de carga, pense en ponerlo en el trigger.

Reply:
`$table->boolean('estado')->default(1)->change();`
`Unknown database type bit requested, Doctrine\DBAL\Platforms\MySqlPlatform may not support it.`
voy a volver a buscar si hay solución

Reply:
Si se modificaba el tipo `bit` a `tinyint`, había que actualizar las consultas en el sistema anterior ya que están utilizando la sentencia `NOT` para calcular el valor binario inverso. Nos conviene dejar los triggers como están, para no tener que actualizar el sistema anterior